index.html

```diff
@@ -0,0 +1,101 @@
+          </a>
+        </div>
+      </div>
+    </header>
+    <section id="headline">
+      <div class="wrapper">
+        <div class="headline-inner">
+          <h1 class="headline-primary">
+            <span class="headline-primary-first">Hey there. I’m Mariane</span>
+            <span class="headline-primary-second"
+              >I’m a software developer</span
+            >
+          </h1>
+          <p class="headline-text">
+            I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a
+            project you need coded, don’t hestiate to contact me.
+          </p>
+          <ul class="headline-social-icons">
+            <li class="headline-social-icon">
+              <a
+                href="https://twitter.com/shahadat3669"
+                class="headline-social-icon-link">
+                <img
+                  src="./dist/images/social-icons/twitter.svg"
+                  alt="twitter icon"
+                  class="headline-social-icon-img" />
+              </a>
+            </li>
+            <li class="headline-social-icon">
+              <a
+                href="https://linkedin.com/in/shahadat3669"
+                class="headline-social-icon-link">
+                <img
+                  src="./dist/images/social-icons/Linkedin.svg"
+                  alt="linkedin icon"
+                  class="headline-social-icon-img" />
+              </a>
+            </li>
+            <li class="headline-social-icon">
+              <a
+                href="https://medium.com/@shahadat3669"
+                class="headline-social-icon-link">
+                <img
+                  src="./dist/images/social-icons/medium.svg"
+                  alt="medium icon"
+                  class="headline-social-icon-img" />
+              </a>
+            </li>
+            <li class="headline-social-icon">
+              <a
+                href="https://github.com/shahadat3669"
+                class="headline-social-icon-link">
+                <img
+                  src="./dist/images/social-icons/github.svg"
+                  alt="github icon"
+                  class="headline-social-icon-img" />
+              </a>
+            </li>
+            <li class="headline-social-icon">
+              <a href="https://angel.co/" class="headline-social-icon-link">
+                <img
+                  src="./dist/images/social-icons/angellist.svg"
+                  alt="angellist icon"
+                  class="headline-social-icon-img" />
+              </a>
+            </li>
+          </ul>
+          <a href="#recent-work" class="headline-scroll">
+            <img
+              src="./dist/images/icon-down.svg"
+              alt="headline scroll icon"
+              class="headline-scroll-icon" />
+          </a>
+        </div>
+      </div>
+    </section>
```

Comment:
- [ ] Good job using the correct semantic tag for your headline section. I have realized that there is a background image that is present in the original design that is currently missing in your design. I would urge you to include it.

**This is how I am seeing this section:**

![image](https://user-images.githubusercontent.com/86318284/198334409-c3cb59ea-9dbc-43b9-8f4e-dc93fd80469b.png)

**This is the desired design with the background images:**

![image](https://user-images.githubusercontent.com/86318284/198334575-fd708e3c-c3fd-4d2d-b9fa-d85b3020594e.png)

**Note:** the difference in the designs is only with the background images. Kindly include them.